Source/Locales/pt-BR/messages.json

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
This file is good. But why did you include it in pull request #131, which is about a favicon image??

@AnandKumar22 , you need to learn how to organize your pull requests...